### PR TITLE
Feature: convert images

### PR DESCRIPTION
**Pull request by DilanOjeda**
I've added a new class Converter and a new feature for converting image. In addition, I've refactored the VideoConverter class code and added new parameters.

### CODE REVIEW

File: CONVERTER-SERVICE/src/models/converter.model.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    }
+  }
+  
+  // Converts files 
+  convert() {
+    throw new Error('Error: convert() must be implemented');
+  }
+}
+
+module.exports = Converter;
```
Comment (stospina) [inline CONVERTER-SERVICE/src/models/converter.model.js]: Add a new line at the end of the code

File: CONVERTER-SERVICE/src/models/imageConverter.model.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    .then( data => {
+      return {response: true, data};
+    })
+    .catch( error => {
+      return {response: false, error};
+    });
+  }
+}
+
+module.exports = ImageConverter;
```
Comment (stospina) [inline CONVERTER-SERVICE/src/models/imageConverter.model.js]: Add a new line at the end of the code

**Review by stospina — COMMENTED** [review]
the new line at the end of file

**Review by stospina — APPROVED** [review]
Everything looks good

File: CONVERTER-SERVICE/src/models/converter.model.js (hunk 1 above)
Comment (DilanOjeda) [reply CONVERTER-SERVICE/src/models/converter.model.js]: It's already fixed

File: CONVERTER-SERVICE/src/models/imageConverter.model.js (hunk 2 above)
Comment (DilanOjeda) [reply CONVERTER-SERVICE/src/models/imageConverter.model.js]: It's already fixed

**Pull request merged**